preprocessors/semanticSeg/segment.py

```diff
@@ -73,7 +73,7 @@ def findContour(pred_color, width, height):
     ret, thresh = cv2.threshold(gray_image, 10, 255, cv2.THRESH_BINARY)
     contours, hierarchy = cv2.findContours(
```

Comment:
What are the failure cases for this? Should this be in a try/catch? If this throws an error, what happens? In general, there seems to be minimal error checking, so if opencv or any of the pytorch calls throw an error, what happens?

Reply:
@jeffbl if there are no contours in the image then the "contours" variable will be a empty list. In that case the for loop would not be iterated and we would simple get an empty list. I don't think there is a need for try catch as the code internally handles that use case.

Reply:
ok. I'm trying to make sure that if there is a failure in any of these calls, that we'll get a meaningful message that will help us track it down. e.g., if it runs out of memory, if the input data is bad somehow, etc. If you're confident that an error will be logged if something goes wrong here, then all good.